File: VB/ReportGallerySample/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
' ...

Namespace ReportGallerySample
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btShowDesigner_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btShowDesigner.Click
			Dim report As New XtraReport1()
			report.ShowDesignerDialog()
		End Sub
	End Class
End Namespace

File: CS/ReportGallerySample/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;
// ...

namespace ReportGallerySample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void btShowDesigner_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            ReportDesignTool designTool = new ReportDesignTool(report);
            designTool.ShowDesignerDialog();
        }
    }
}
